ci-operator/config/openshift/verification-tests/openshift-verification-tests-master.yaml

```diff
@@ -0,0 +1,25 @@
+build_root:
+  image_stream_tag:
+    cluster: 'https://api.ci.openshift.org'
+    name: 'bushslicer'
+    namespace: 'ci'
+    tag: 'centos'
+images:
+- to: centos
+  dockerfile_path: tools/openshift-ci/Dockerfile.centos
+promotion:
+  name: 'bushslicer'
```

Comment:
might want to rename?

Reply:
I can recreate the imagestream with the correct name.